plugins/terminal/eos.py

```diff
@@ -52,6 +52,17 @@ class TerminalModule(TerminalBase):
         re.compile(br"% Subnet [0-9a-f.:/]+ overlaps", re.I),
         re.compile(br"Maximum number of pending sessions has been reached"),
         re.compile(br"% Prefix length must be less than"),
+        # terminal error when a configured channel-group's mode is modified.
+        # 
+        # veos(config-if-Et2)#show active 
+        # interface Ethernet2
+        # channel-group 15 mode on
+        # -----------------------------
+        # veos(config-if-Et2)#channel-group 15 mode active
+        # % Cannot change mode; remove all members and try again.
```

Comment:
```suggestion
        # May be returned in response to `channel-group <number> mode <mode>`
```
This is all I'm really after. I just want stderr entries to have some indication of what causes them so we can have a chance to ever audit them some day down the road.